cmd/stacker/lxc-wrapper/lxc-wrapper.c

```diff
@@ -253,16 +253,19 @@ int main(int argc, char *argv[])
 	if (!strcmp(argv[1], "spawn")) {
 		int ret, status;
 		char *name, *lxcpath, *config_path;
+		char **args = NULL;
 
-		if (argc != 5) {
+		if (argc < 5) {
 			fprintf(stderr, "bad number of args for spawn: %d\n", argc);
 			return 1;
 		}
 
-
 		name = argv[2];
 		lxcpath = argv[3];
 		config_path = argv[4];
+		if (argc >= 5) {
+			args = &argv[5];
+		}
```

Comment:
@hallyn ,

I wasn't sure if I could actually do this or not.  I'm just passing 'spawns' argv into spawn_container, which ultimately calls 'start'.

do i need to create my own copy of argv[5...] ?